pyproject.toml

```diff
@@ -186,3 +187,10 @@ source = "vcs"
 [build-system]
 requires = ["hatchling", "hatch-vcs"]
 build-backend = "hatchling.build"
+
+[tool.codespell]
+# Ref: https://github.com/codespell-project/codespell#using-a-config-file
+skip = '.git*,*.svg,*.lock,*.css,pyproject.toml,.mypy_cache,.venv'
+check-hidden = true
+ignore-regex = '\bla vie\b'
+ignore-words-list = 'ans'
```

Comment:
Just a cosmetic request: Could you please move the `[tool.codespell]` section further up, e.g. after the `[tool.commitizen]` section, so these `[tool.*]` sections are grouped? `[tool.hatch.*]` is a bit special because it's related to `[build-system]`, so I'd keep those two close.